src/main/java/io/vlingo/wire/fdx/bidirectional/rsocket/RSocketClientChannel.java

```diff
@@ -143,27 +144,26 @@ private void prepareChannel() {
   private static class ChannelResponseHandler {
     private final ResponseChannelConsumer consumer;
     private final Logger logger;
-    private final ByteBufferPool readBufferPool;
+    private final ConsumerByteBufferPool readBufferPool;
 
     private ChannelResponseHandler(final ResponseChannelConsumer consumer, final int maxBufferPoolSize, final int maxMessageSize, final Logger logger) {
       this.consumer = consumer;
-      this.readBufferPool = new ByteBufferPool(maxBufferPoolSize, maxMessageSize);
+      this.readBufferPool = new ConsumerByteBufferPool(
+          ElasticResourcePool.Config.of(maxBufferPoolSize), maxMessageSize);
       this.logger = logger;
     }
 
     private void handle(Payload payload) {
-      final ByteBufferPool.PooledByteBuffer pooledBuffer = readBufferPool.accessFor("client-response", 25);
+      final ConsumerByteBuffer pooledBuffer = readBufferPool.acquire();
       try {
         final ByteBuffer payloadData = payload.getData();
         final ConsumerByteBuffer put = pooledBuffer.put(payloadData);
         consumer.consume(put.flip());
       } catch (final Throwable e) {
```

Comment:
Why is it ok to catch and just log `Throwable` here?

Reply:
I haven't actually. There are several places in wire where `Throwable` is caught instead of `Exception`. Should I change it as part of this PR?

Reply:
Yes, please do.